src/database/migrations/20180517100530_create_locales.js

```diff
@@ -0,0 +1,57 @@
+import localesJson from '../seeds/locales.json'

```

Comment:
I guess we could have a data folder in the migrations and put the locales there? Since it’s not actually a seed, as you pointed out.